doc/release/upcoming_changes/16841.changes.rst

```diff
@@ -0,0 +1,4 @@
+Output if ``dtype`` of ``linspace`` is int
+----------------------------------------------------
+- If the ``dtype`` of the ``linspace`` is int, round down it and casted it.
```

Comment:
I don't understand this note after the comma, in particular what is it? The end point?